### ISSUE TITLE
No Automatic Post-Processing

### ISSUE THREAD

**Issue by Yellow1144** [SiCKRAGETV/sickrage-issues#2539]
Sorry.. Issue title should really read "No Automatic Post-Processing" CAn't seem to change it.

Since the last time I updated SR no longer Post Processes my incoming folder. It was working perfectly before the last update. 

PP folder and destination folders are on different drives, nothing has changed on the system at all. My logging is always set to debug mode and post-processing does not show up in the log at all until i manually execute it. There are no errors or problematic notices in the log at all

Manual PP works fine and is the only way that I am able to get my stuff post processed

BRANCH: (master) / COMMIT: (99ecfb5edcae4a8e5facff73f39ff1415a5e3e96) 
Win 7 Pro/64
Python: 2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]

**Comment by neoatomic** [comment]
@Yellow1144 Is the setting `Scan and Process` enabled under Post Processing.? And have you set the time under `Auto Post-Processing Frequency` ?

**Comment by WebSpider** [comment]
Also, please consider updating. SiCKRAGETV/SickRage@99ecfb5edcae4a8e5facff73f39ff1415a5e3e96 is 2 weeks old

**Comment by Yellow1144** [comment]
Yes, those settings are correct and are as they were before. I also tried deselecting that option, saving, then re-selecting and saving again. Frequency is as it was at 10 minutes.

There is no way I am updating giving the huge number of issues I am seeing being reported here after the latest round of updates. In fact I am thinking of reverting to an older backed up version that was working perfectly. In my opinion the development of this great app is way too chaotic, with too many changes being made quickly with seemingly no testing before release. I never automatically update. If I did, I would be spending far too much time screwing around trying to fix things rather than just using the app. I'll just watch the activity and update when things have calmed  down for a while.
